xml/System.Runtime.InteropServices/DllImportResolver.xml

```diff
@@ -39,7 +39,7 @@
 
 The runtime does not do any lifetime management around the handle returned by a <xref:System.Runtime.InteropServices.DllImportResolver>. It is left to the implementation and consuming code to keep the library loaded for as long as necessary and free it if/when desired. 
 
-For each entry point, the resolver is called the first time PInvoke is used. The implementation of the resolver may want to cache the `libraryName` to handle mapping to improve performance as long as the library isn't unloaded via <xref:System.Runtime.InteropServices.NativeLibrary.Free>.
+For each entry point, the resolver is called the first time PInvoke is used. To improve performance, the implementation of the resolver can cache the `libraryName` to handle mapping, as long as the library isn't unloaded via <xref:System.Runtime.InteropServices.NativeLibrary.Free(System.IntPtr)>.
```

Comment:
> the first time PInvoke is used.

Rereading this, it doesn't sound right to me. In my reading it is stating "PInvoke" is a singlular instance, but which one? Should this instead be "the first time a PInvoke is used."?